editoast/src/views/infra/railjson.rs

```diff
@@ -0,0 +1,83 @@
+use crate::api_error::{ApiError, ApiResult};
+
+use crate::db_connection::DBConnection;
+use crate::infra::RAILJSON_VERSION;
+use crate::schema::{RailJson, RailjsonQueryable};
+use diesel::sql_types::Integer;
+use diesel::{sql_query, RunQueryDsl};
+use rocket::http::Status;
+use rocket::response::status::Custom;
+use rocket::serde::json::{Error as JsonError, Json, Value as JsonValue};
+
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+enum ListErrorsRailjson {
+    #[error("Wrong Railjson version provided")]
+    WrongRailjsonVersionProvided,
+}
+
+impl ApiError for ListErrorsRailjson {
+    fn get_status(&self) -> Status {
+        Status::BadRequest
+    }
+
+    fn get_type(&self) -> &'static str {
+        match self {
+            ListErrorsRailjson::WrongRailjsonVersionProvided => {
+                "editoast:infra:railjson:WrongRailjsonVersionProvided"
+            }
+        }
+    }
+}
+/// Serialize an infra
+#[get("/<infra>/railjson?<exclude_extensions>")]
+pub async fn get_railjson(
+    infra: i32,
+    exclude_extensions: bool,
+    conn: DBConnection,
+) -> ApiResult<Custom<JsonValue>> {
```

Comment:
We can let postgres serialize the infra.
```suggestion
) -> ApiResult<Custom<String>> {
```